File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution{
    public boolean hasDuplicate(int[] nums){
        for (int i = 0; i < nums.length; i += 1){
            for (int j = i+1; j < nums.length; j += 1){
                if (nums[i] == nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:

        nums.sort()

        #result = [1] * len(nums)
        seen = {} # key is num, value is longest subsequence
        for i in range(len(nums)):
            if i == 0:
                seen[nums[i]] = 1
                continue
            if (nums[i] - 1) in seen:
                seen[nums[i]] = seen.get(nums[i] - 1) + 1
            else:
                seen[nums[i]] = 1
        print(seen)
        if seen:
            return max(seen.values())
        return 0